src/art.py

```
@@ -32,18 +29,35 @@ def get_random_art() -> str:
+        try:
+            from openai import AsyncOpenAI
+            client = AsyncOpenAI(api_key=api_key)
+            response = await client.images.generate(
+                model="gpt-image-1",
+                prompt=prompt,
+                size="1792x1024",
+                quality="auto",
+                n=1,
+            )
+            return response.data[0].url
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)